File: file-upload/src/main/java/com/reservation/fileupload/FileUploadApplication.java
package com.reservation.fileupload;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = {
	"com.reservation.auth",
	"com.reservation.domain",
	"com.reservation.fileupload",
})
public class FileUploadApplication {
	public static void main(String[] args) {
		SpringApplication.run(FileUploadApplication.class, args);
	}
}

File: file-upload/src/main/kotlin/com/reservation/fileupload/FileUploadApplication.kt
package com.reservation.fileupload

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication(
    scanBasePackages = ["com.reservation.auth", "com.reservation.domain", "com.reservation.fileupload"
    ]
)
open class FileUploadApplication {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(FileUploadApplication::class.java, *args)
        }
    }
}
